modules/services/repositories/src/test/java/au/com/shiftyjelly/pocketcasts/repositories/shownotes/ShowNotesManagerTest.kt

```diff
@@ -0,0 +1,83 @@
+package au.com.shiftyjelly.pocketcasts.repositories.shownotes
+
+import au.com.shiftyjelly.pocketcasts.repositories.podcast.EpisodeManager
+import au.com.shiftyjelly.pocketcasts.repositories.podcast.ImageUrlUpdate
+import au.com.shiftyjelly.pocketcasts.servers.ServerShowNotesManager
+import au.com.shiftyjelly.pocketcasts.servers.podcast.ShowNotesEpisode
+import au.com.shiftyjelly.pocketcasts.servers.podcast.ShowNotesPodcast
+import au.com.shiftyjelly.pocketcasts.servers.podcast.ShowNotesResponse
+import au.com.shiftyjelly.pocketcasts.sharedtest.MainCoroutineRule
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.test.runTest
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.Mock
+import org.mockito.junit.MockitoJUnitRunner
+import org.mockito.kotlin.verify
+
+@OptIn(ExperimentalCoroutinesApi::class)
+@RunWith(MockitoJUnitRunner::class)
+class ShowNotesManagerTest {
+    @get:Rule
+    val coroutineRule = MainCoroutineRule()
+
+    @Mock lateinit var serverShowNotesManager: ServerShowNotesManager
+    @Mock lateinit var episodeManager: EpisodeManager
+
+    private lateinit var showNotesManager: ShowNotesManager
+
+    @Before
+    fun setup() {
+        showNotesManager = ShowNotesManager(
+            serverShowNotesManager = serverShowNotesManager,
+            episodeManager = episodeManager,
+        )
+    }
+
+    @Test
+    fun test() = runTest {
```

Comment:
So nice to see tests being added here!

Nitpick: Can we give it a more meaningful name?

Reply:
Thanks! I obviously forgot to update that name. 🤦